certs/generate-cert.sh

```diff
@@ -0,0 +1,33 @@
+PARENT="localhost"
+openssl req \
+-x509 \
+-newkey rsa:4096 \
+-sha256 \
+-days 365 \
```

Comment:
it's better to have a warning in the doc, saying, "in every 365 days, you have to rotate/recreate the certs"